### ISSUE TITLE
[feature request] - Adding interface to include location data from other moving object in the scheduling

### ISSUE THREAD

**Issue by m2joest** [VDA5050/VDA5050#50]
For an holistic traffic management it would be great to incorporate also location data of other moving objects (pallets, forklifts, people) in the scheduling | planing process

**Comment by AntonDueck** [comment]
the VDA5050 protocol is a interface between the AGV and MasterControl, you could use fake AGV for you palles/objeckts or use an other channel fot this information

**Comment by m2joest** [comment]
Yes. Having an data feed based on the open-locating standard  can been used, like some vendors already do. But in principle it would be great having the master control beeing able to distribute order also to non-AGVs (like fork lifts or even humans)

**Comment by ConstantinUp** [comment]
In theory, this is already possible by having a vehicle or other device that only reports a state and does not receive orders or instant actions. After the release of version 3.0, we plan to add an optional feature for this use case, such as a separate visualization channel. If you have an idea for how to easily incorporate this, please create a pull request against the dev/V3.0.0 branch.
